src/app/pages/wallets/wallet-details/transaction-editor/pages-wallet-transaction-editor.component.html

```diff
@@ -0,0 +1,36 @@
+<form [formGroup]='form'>
+  <h1 mat-dialog-title>{{ data ? "Update Transaction" : "Create Transaction" }}</h1>
+  <mat-form-field appearance="fill">
+    <mat-label>Select type</mat-label>
+    <mat-select [formControl]='form.controls["type"]'>
+      <mat-option *ngFor="let type of selectTransactionsTypes | keyvalue" [value]="type.value">
+        {{ type.key }}
+      </mat-option>
+    </mat-select>
+  </mat-form-field>
+  <mat-form-field *ngIf='form.controls["description"]' appearance="fill">
+    <mat-label>Transaction description(optional)</mat-label>
+    <input matInput [formControl]='form.controls["description"]'>
+  </mat-form-field>
+  <mat-form-field appearance="fill">
+    <mat-label>Transaction amount</mat-label>
+    <input matInput [formControl]='form.controls["amount"]'>
+    <mat-error *ngIf="amountIsNotProvided">
+      Amount is required
+    </mat-error>
+    <mat-error *ngIf="amountHasInvalidFormat">
+      Amount has invalid format
+    </mat-error>
+  </mat-form-field>
+  <mat-form-field class="example-full-width" appearance="fill">
+    <mat-label>Choose a date</mat-label>
+    <input matInput [formControl]='form.controls["date"]' [matDatepicker]="picker">
```

Comment:
Please also add some validation for this one:
- required
- matching `DD/MM/YYYY` format